src/fs/fs.go

```diff
@@ -113,3 +113,13 @@ func WriteFile(fromFile io.Reader, to string, mode os.FileMode) error {
 	// And move it to its final destination.
 	return os.Rename(tempFile.Name(), to)
 }
+
+// IsDirectory checks if a given path is a directory
+func IsDirectory(path string) bool {
+	info, err := os.Stat(path)
+	if err != nil {
```

Comment:
FYI you could one line this:
```
return err == nil && info.IsDir()
```
Not 100% sure about the case where `err != nil` but I guess there's not much to be done.

Reply:
I think it should be fine with that one liner, if it's err != nil, we'd return false anyways